pkg/cidata/cidata.TEMPLATE.d/boot/05-lima-volumes.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+test $LIMA_CIDATA_VOLUMES -gt 0 || exit 0
+
+get_volume_var () {
+  volumevarname="LIMA_CIDATA_VOLUME_${1}_${2}"
+  echo "$(eval echo \$"$volumevarname")"
+}
+
+for i in $(seq 0 $((LIMA_CIDATA_VOLUMES - 1))); do
+  VOLUME_NAME="$(get_volume_var $i "NAME")"
+  DEVICE_NAME="$(get_volume_var $i "DEVICE")"
+
+  # first time setup
+  if [[ ! -b "/dev/disk/by-label/lima-${VOLUME_NAME}" ]]; then
+    # TODO: skip if volume is tagged as "raw"
+    echo 'type=83' | sfdisk --label dos "/dev/${DEVICE_NAME}"
```

Comment:
Please add a comment line to explain this magic value

Reply:
I think I'll just replace the magic value with the `linux` alias as per the [manpage](https://man7.org/linux/man-pages/man8/sfdisk.8.html), any concerns?